### DISCUSSION TITLE
吐槽性能问题严重

### DISCUSSION THREAD

**Discussion by liush6** [the1812/Bilibili-Evolved#3184] [综合讨论 / General]
单启用此脚本会使得CPU占用过高，笔记本9750+1660t的配置，刷新页面，在一些情况下（偶尔）直接占用100%，一般也是50 60%的占用。而启用adguard后，（adguard过滤器拦截B站视频的话，关闭几个关键的过滤条目就能正常使用）两者同时使用就会在看个视频后，再刷新就会让CPU满载，页面也变得缓慢，滚动与控件调用也很卡。

**Comment by the1812** [comment]
背景模糊相关的功能关掉
